tram-one.js

```diff
@@ -31,6 +31,17 @@ class Tram {
     return this
   }
 
+  addListener(listener) {
+    assert.equal(
+      typeof listener, 'object',
+      'Tram-One: listener should be { store-key: { listener-name: listener-function } }'
+    )
```

Comment:
the assert should be of type function, you can look at the addRoute method for an example of that.